1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb

```diff
@@ -1023,6 +1034,13 @@
     "    + `wb.data.DataFrame` returned a Pandas dataframe, which is very convenient for further data analysis\n",
     "- Many client libraries do much more than just retrieving the API results"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Final note:** Remember to go to `File` > `Save a copy in Drive` to save a copy of this notebook in your Google account."
```

Comment:
Not sure we still want to say this now that we are running this session in databricks. For what is worth, to save the notebook it's `File > Export > IPython Notebook` in databricks

Reply:
added clarifying note in 09b9665